mirgecom/steppers.py

```diff
@@ -232,6 +230,7 @@ def _advance_state_leap(rhs, timestepper, state,
                     state, dt = post_step_callback(state=state,
                                                    step=istep,
                                                    t=t, dt=dt)
+                stepper_cls.dt = dt
```

Comment:
I think this can/should be moved into the if-block directly above it. The timestep the stepper class has only needs to be updated if dt is changed.

Reply:
(68e1e60)